cob_description/urdf/sensors/usb_cam.gazebo.xacro

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:macro name="usb_cam_gazebo" params="name ros_topic">
+    <gazebo reference="${name}_frame">
+
+    <sensor type="camera" name="${name}">
+      <update_rate>30.0</update_rate>
```

Comment:
framerate for real camera is 5hz, see https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_bringup/drivers/usb_camera_node.launch#L14